sequelize/index.d.ts

```diff
@@ -2,12 +2,11 @@
 // Project: http://sequelizejs.com
 // Definitions by: samuelneff <https://github.com/samuelneff>, Peter Harris <https://github.com/codeanimal>, Ivan Drinchev <https://github.com/drinchev>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-
 // Based on original work by: samuelneff <https://github.com/samuelneff/sequelize-auto-ts/blob/master/lib/sequelize.d.ts>
-
 /// <reference types="lodash" />
 /// <reference types="bluebird" />
 /// <reference types="validator" />
+// TypeScript Version: 2.1
```

Comment:
I think I tried also that way... is it OK if there are comments after that line?